### PR TITLE
Add an option for actionbar hit cooldown

### PR DESCRIPTION
**Pull request by ImDaBigBoss**
In the config, you can set `actionbar-cooldown` to true so that the hit cooldown shows up in the actionbar instead of in the title.

### CODE REVIEW

**Comment by ImDaBigBoss** [comment]
This fixes #1685

File: connector/src/main/java/org/geysermc/connector/utils/CooldownUtils.java [hunk 1]
```diff
@@ -67,7 +72,10 @@ private static void computeCooldown(GeyserSession session, long lastHitTime) {
         if (session.isClosed()) return; // Don't run scheduled tasks if the client left
         if (lastHitTime != session.getLastHitTime()) return; // Means another cooldown has started so there's no need to continue this one
         SetTitlePacket titlePacket = new SetTitlePacket();
-        titlePacket.setType(SetTitlePacket.Type.SUBTITLE);
+        if (ACTIONBAR_COOLDOWN)
```
Comment (Camotoy) [inline connector/src/main/java/org/geysermc/connector/utils/CooldownUtils.java:75]: Please use brackets here and for the else statement.

**Comment by Camotoy** [comment]
Can you merge master into this so your build can compile? :)

**Comment by Camotoy** [comment]
The concept looks fine to me, but we're internally discussing if we want this change merged with the current configuration option - this would break existing configs most likely though. Just letting you know so we're not just ignoring your PR!

**Comment by Camotoy** [comment]
I'm so sorry on keeping you waiting! Here's what I propose as a better alternative to two configuration options:
- keep everything under `show-cooldown`
- `false` still means no cooldown should be shown. `true` should redirect to `title`.
- `actionbar` as the input means that the actionbar should be used.

Does that sound good? Let me know if you need help!

**Comment by ImDaBigBoss** [comment]
@Camotoy, I have done what you requested, if you need anything else, I'm available to do it.

**Comment by Camotoy** [comment]
Looks like your mappings and languages submodules need to be updated!

**Comment by ImDaBigBoss** [comment]
How can I do that?

**Comment by Camotoy** [comment]
`git submodule update`
`git add connector/src/main/resources/mappings`
`git add connector/src/main/resources/languages`
`git commit -m "Update mappings"`

Should do the trick?

**Comment by ImDaBigBoss** [comment]
It says I'm up to date and that `connector/src/main/resources/languages` is untracked content

**Comment by ImDaBigBoss** [comment]
I don't understand, it refuses to update the mappings.

**Comment by ImDaBigBoss** [comment]
It's fixed! Though I'm not sure what I did to do so...

**Comment by Camotoy** [comment]
Do you think you could make an enum for each option so we don't have to compare a string each time? See https://github.com/GeyserMC/Geyser/blob/master/connector/src/main/java/org/geysermc/connector/common/AuthType.java#L49 as an example of what I'm talking about.

Also, I meant that `show-cooldown`'s title option should literally be called `title` in the config and `true` should also equal title.

File: connector/src/main/resources/config.yml [hunk 2]
```diff
@@ -112,6 +112,7 @@ allow-third-party-capes: true
 allow-third-party-ears: false
 
 # Allow a fake cooldown indicator to be sent. Bedrock players do not see a cooldown as they still use 1.8 combat
+# Can be true, false or actionbar to show in the actionbar.
 show-cooldown: true
```
Comment (Camotoy) [inline connector/src/main/resources/config.yml:116]: Update this and the above message to `title`.

**Review by Camotoy — APPROVED** [review]

**Comment by Camotoy** [comment]
Thank you for your contribution, and thank you for being patient!

**Pull request merged**